ClassTranscribeServer/Controllers/CaptionsSearchController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ClassTranscribeDatabase;
+using ClassTranscribeDatabase.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Nest;
+using System;
+using System.Threading.Tasks;
+
+namespace ClassTranscribeServer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CaptionsSearchController : BaseController
+    {
+        private readonly IElasticClient _elasticClient;
+
+        public CaptionsSearchController(CTDbContext context,
+            IElasticClient client,
+            ILogger<CaptionsSearchController> logger) : base(context, logger)
+        {
+            _elasticClient = client;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<Caption>> Search(string index, string keyword)
```

Comment:
Todo: Will we need a security access check implemented here?

Reply:
Yes, I think that will be implemented once we have everything working after the frontend is integrated. Would that be okay?

Reply:
Yes of course! Suggest you at least put a code comment in though "TODO:...."

Reply:
Done